src/main/java/org/broadinstitute/hellbender/utils/read/SAMRecordToGATKReadAdapter.java

```diff
@@ -717,4 +717,12 @@ public void setHeader(SAMFileHeader header) {
 
         samRecord.setHeaderStrict(header);
     }
+
+    public Object getTransientAttribute(Object key) {
+        return samRecord.getTransientAttribute(key);
+    }
+
+    public void setTransientAttribute(Object key, Object value) {
```

Comment:
Add javadoc for these two new methods, with appropriate disclaimers about use of these methods, and an explanation of what the transient attributes are.